index.html

```diff
@@ -1799,7 +1817,126 @@ <h3 id="Vertical_Whitespace" style="left: -46px; position: relative;">Vertical W
+<h2 id="Testing" style="left: -46px; position: relative;">Testing</h2>
+      
+<p>This section contains the style conventions <em>specifically for creating tests</em>. This covers all forms 
+   of tests in the HYPED Codebase, please read this carefully before beginning testing.</p>
+  
+  <!-- Test Names -->
+  <h3 id="Test Names" style="left: -46px; position: relative">Test names</h3>
+
+  <div class="summary">
+    <p>Test names should be written in <em><b>camelCase with the name beginning with a verb and first letter lowercased
+       then following CamelCase afterwards</b></em> 
+       This closely follows the Google primer style guide for names of tests. Which is what a lot
+       of the testing style's are inspired from
+    </p>
+  </div>  
+  <div class="stylebody">
+    <pre class="badcode">// Bad- spaced using underscores, capitalisation is inconsistent
+TEST_F(DifferentiatorFunctionality, handles_zero_Input) {
+...
+}
+    </pre>
+    <pre>// Good - correct spacing convention, capitalisation is consistent
+TEST_F(DifferentiatorFunctionality, handlesZeroInput){
+...
+}
+    </pre>            
+  </div>
+
+<!-- Test File names -->
+<h3 id="Test_File_Names" style="left: -46px; position: relative">Test file names</h3>
+<div class="summary">
+  <p> There is one special rule for the naming of test files. They should all end in .test.cpp </p>
+  <p> NOTE: That is not going to change the file type from a typical .cpp file it is just a naming
+      convention commonly used for testing and allows for people to see instantly that it is a test files
+  </p>
+  <pre class="badcode">// Bad file name
+differentiator_test.cpp
+  </pre>
+  <pre>// Good file name
+differentiator.test.cpp
+  </pre>
+</div>
+<!-- Test Fixtures -->
+  <h3 id="Test Fixtures" style="left: -46px; position: relative">Test Fixtures</h3>
+
+  <div class="summary">
+    <p> We have two styling rules for test fixtures:
+      <ul>
+        <li>Firstly test fixtures <em><b>must be structs</b></em></li>
+        <li>
+            Secondly test fixtures <em><b>must begin with a capital letter</b></em> and then <em><b>follow 
+            camel case styling for capitalisation</b></em></p>
+        </li>
+      </ul>              
+  </div>
+
+  <div class="stylebody">
+    <pre class="badcode">// Bad - wrong capitalisation
+struct differentiatorFunctionality : public ::testing::Test {
+...
+}
+    </pre>
+    <pre>// Good - capitalisation correct
+struct DifferentiatorFunctionality : public ::testing::Test {
+...
+}
+    </pre>
+  </div>
+  
+  <!-- Test Fixture Variables vs Local Test Variable -->
+  <h3 id="Test Fixture Variables vs Local Test Variables" style="left: -46px; position: relative">Test fixture variables vs local test variables</h3>
+  <div class="stylebody">
+    <p> When declaring variables in tests we need to be consistent about where we accessing them from and their scopes. <em><b>All variables which you will use
```

Comment:
Should it read "we need to be consistent about where we **are** accessing them from and their scopes." ?

Reply:
Yes just noticed that formatting error on my end. It will display fine on the actual page though

Reply:
Sorry, what do you mean?

Reply:
nevermind ignore what i said, i was reading it on my phone and the formatting for  the <p> got all messed up due to screen size. Was half asleep at the time haha